cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,281 @@ Cask Data Application Platform Release Notes
+  Added a check to cause a DB (source or sink) pipeline to fail during deployment if the table (source or sink)
+  was not found, or if an incorrect connection string was provided.
+
+- `CDAP-3815<https://issues.cask.co/browse/CDAP-3815>`__ -
+  Fixed an issue where the regex filter for S3 Batch Source wasn't getting applied correctly.
+
+- `CDAP-4000<https://issues.cask.co/browse/CDAP-4000>`__ -
+  Fixed an issue where a stream that has a view could not be deleted cleanly.
+
+- `CDAP-1174<https://issues.cask.co/browse/CDAP-1174>`__ -
+  Prefix start script functions with cdap to prevent namespace collisions
+
+- `CDAP-4678<https://issues.cask.co/browse/CDAP-4678>`__ -
+  Fixed issue where Logs and History list on Hydrator pipeline view is not updating on new runs
+
+- `CDAP-4658<https://issues.cask.co/browse/CDAP-4658>`__ -
+  Added system metadata UI for CDAP entities
+
+- `CDAP-4598<https://issues.cask.co/browse/CDAP-4598>`__ -
+  Adding error message when a required field is not filled up when configuring Hydrator Pipeline
+
+- `CDAP-4083<https://issues.cask.co/browse/CDAP-4083>`__ -
+  Fixed an issue with the documentation of the Data Quality Application's "Unique Value" function.
+
+- `CDAP-4145<https://issues.cask.co/browse/CDAP-4145>`__ -
+  Added documentation of the new view and stream-views feature.
+
+- `CDAP-4141<https://issues.cask.co/browse/CDAP-4141>`__ -
+  Fixed node.js version checking for the cdap.sh script in the cdap sdk.
+
+- `CDAP-4393<https://issues.cask.co/browse/CDAP-4393>`__ -
+  Fixed the bug in Tags search which didnt' allow user to search for all the tags.
+
+- `CDAP-3695<https://issues.cask.co/browse/CDAP-3695>`__ -
+  No failures occurred in the recent builds. Marking as resolved fix.
+
+- `CDAP-4301<https://issues.cask.co/browse/CDAP-4301>`__ -
+  Added support for CDH5.5
+
+- `CDAP-3900<https://issues.cask.co/browse/CDAP-3900>`__ -
+  Return empty list in case of logs are requested for the deleted program runs.
+
+- `CDAP-3542<https://issues.cask.co/browse/CDAP-3542>`__ -
+  Fixed issue when workflow contains fork and program executing on one of the branches fails because of some error,
+  other concurrently running programs were remaining in the RUNNING state, even if the Workflow is completed.
+
+- `CDAP-4119<https://issues.cask.co/browse/CDAP-4119>`__ -
+  When CDAP master switches from active to standby, the programs that are running are marked failed.
+
+- `CDAP-3345<https://issues.cask.co/browse/CDAP-3345>`__ -
+  Fixed a bug where TTL for datasets were incorrect, reduced by (/1000) after upgrade. After running upgradeTool,
+  please make sure the TTL values of tables are as expected.
+
+- `CDAP-3861<https://issues.cask.co/browse/CDAP-3861>`__ -
+  Fixed an issue about stopping all dependent services when a service is stopped.
+
+- `CDAP-2878<https://issues.cask.co/browse/CDAP-2878>`__ -
+  [Note: Please remove this JIRA from known-issues in Release notes.]
```

Comment:
oh perhaps this is a pointer that we should remove this jira from existing release notes? not sure.